Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -1423,69 +1346,68 @@ void EmbeddedFrag::randomSampleFlipsAndPermutations(
     nd4 = deg4nodes.size();
   }
 
-  nt = nb + nd4;
+  unsigned int nt = nb + nd4;
```

Comment:
auto?

Reply:
tried that: ends up being a size_t and producing warnings later.